CDP4IME/ViewModels/PluginInstallerViewModel.cs

```diff
@@ -0,0 +1,216 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="PluginInstallerViewModel.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2020 RHEA System S.A.
+//
+//    Author: Sam Gerené, Alex Vorobiev, Alexander van Delft, Nathanael Smiechowski, Kamil Wojnowski
+//
+//    This file is part of CDP4-Plugin Installer Community Edition. 
+//    The CDP4-Plugin Installer Community Edition is the RHEA Plugin Installer for the CDP4-IME Community Edition.
+//
+//    The CDP4-Plugin Installer Community Edition is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Affero General Public
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or any later version.
+//
+//    The CDP4-Plugin Installer Community Edition is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//    GNU Affero General Public License for more details.
+//
+//    You should have received a copy of the GNU Affero General Public License
+//    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace CDP4IME.ViewModels
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using System.Windows.Threading;
+
+    using CDP4Composition.Modularity;
+    using CDP4Composition.ViewModels;
+
+    using CDP4IME.Behaviors;
+    using CDP4IME.Views;
+
+    using NLog;
+
+    using ReactiveUI;
+
+    /// <summary>
+    /// The <see cref="PluginInstallerViewModel"/> is the view model of the <see cref="PluginInstallerViewModel"/> holding it properties its properties and interaction logic
+    /// </summary>
+    public class PluginInstallerViewModel : ReactiveObject, IPluginInstallerViewModel
+    {
+        /// <summary>
+        /// Backing field for the <see cref="ThereIsNoInstallationInProgress"/> property
+        /// </summary>
+        private bool thereIsNoInstallationInProgress = true;
```

Comment:
I think we should never name variablenames "negatively". 
It's just a little harder to read.
So this should be `IsInstallationInProgress isInstallationInProgress IsInstallationInProgressObservable` and value interpretation should of course be reversed.